winlogbeat/sys/wineventlog/format_message.go

```diff
@@ -76,7 +76,7 @@ func evtFormatMessage(metadataHandle EvtHandle, eventHandle EvtHandle, messageID
 	}
 
 	// best guess render buffer size, 16KB, to avoid rendering message twice in most cases
-	const bestGuessRenderBufferSize = 1 << 14
+	const bestGuessRenderBufferSize = 19 // 512KB, 256K wide characters
```

Comment:
```suggestion
	const bestGuessRenderBufferSize = 1 << 19 // 512KB, 256K wide characters
```

also fix the comment in https://github.com/elastic/beats/pull/41327/files#diff-a7b20f94a10542cc7280d552c361526c42d2c8020806fb53ebc910a653aa9551R78